src/test/scala/ai/privado/languageEngine/kotlin/tagger/sink/framework/retrofit/RetrofitAPITaggerTest.scala

```diff
@@ -0,0 +1,83 @@
+package ai.privado.languageEngine.kotlin.tagger.sink.framework.retrofit
+
+import ai.privado.testfixtures.KotlinFrontendTestSuite
+import io.shiftleft.semanticcpg.language.*
+import ai.privado.cache.RuleCache
+import ai.privado.model.*
+import ai.privado.rule.RuleInfoTestData
+import ai.privado.tagger.sink.api.APIValidator
+import ai.privado.entrypoint.PrivadoInput
+import ai.privado.model.{CatLevelOne, Constants, SourceCodeModel}
+
+class RetrofitAPITaggerTest extends KotlinFrontendTestSuite {
+
+  val collectionRule = List(
```

Comment:
Is there a base class where these rules could be added? If these are not specific to this test, can we move them there?